tests/rates/staticRates.spec.ts

```diff
@@ -99,4 +99,91 @@ describe("Rate tests", () => {
     expect(amounts[0].buy).toEqual(trades[0].buy);
     expect(amounts[1].sell).toEqual(trades[1].sell);
   });
+
+  const payAsset = new Asset(
+    "PAY",
+    Ledger.Ethereum,
+    "0xB97048628DB6B661D4C2aA833e95Dbe1A905B280"
+  );
+
+  it("Should consider the rate profitable when the proposed rate is less than the configured rate and one asset is ERC20", async () => {
+    const config = {
+      PAY: { bitcoin: 0.001 },
+      bitcoin: { PAY: 0.01 }
+    };
+
+    const trade = {
+      timestamp: new Date(),
+      buy: {
+        ledger: Ledger.Bitcoin,
+        asset: Asset.bitcoin,
+        quantity: new Big(0.1)
+      },
+      sell: {
+        ledger: Ledger.Ethereum,
+        asset: payAsset,
+        quantity: new Big(0.0009)
+      },
+      protocol: "rfc003"
+    };
+
+    const rates = new StaticRates(config);
+    expect(await rates.isTradeAcceptable(trade)).toBeTruthy();
+  });
+
+  it("Should consider the rate NOT profitable when the proposed rate is greater than the configured rate and one asset is ERC20", async () => {
+    const config = {
+      PAY: { bitcoin: 0.001 },
+      bitcoin: { PAg: 0.01 }
```

Comment:
```suggestion
      bitcoin: { PAY: 0.01 }
```
interesting that it accepted that :)

Reply:
It's because the rate used is the other one.